reference/commands/cache.rst

```diff
@@ -158,26 +158,53 @@ conan cache clean
 .. code-block:: text
 
     $ conan cache clean -h
-    usage: conan cache clean [-h] [-v [V]] [-s] [-b] [-d]
-                             [-p PACKAGE_QUERY]
-                             pattern
+    usage: conan cache clean [-h] [-v [V]] [-s] [-b] [-d] [-t] [-p PACKAGE_QUERY]
+                             [pattern]
 
     Remove non-critical folders from the cache, like source, build and/or download
     (.tgz store) ones.
 
     positional arguments:
       pattern               Selection pattern for references to clean
 
-    optional arguments:
+    options:
```

Comment:
But that's not the real output of the command, is it?

Reply:
I think it depends on the OS Python distro, the message is different